ts/screens/ErrorScreen.tsx

```diff
@@ -0,0 +1,79 @@
+import { Body, Button, Container, H1, Left, Text, View } from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { connect } from "react-redux";
+import AppHeader from "../components/ui/AppHeader";
+import IconFont from "../components/ui/IconFont";
+import { ReduxProps } from "../store/actions/types";
+
+type OwnProps = {
+  navigation: NavigationScreenProp<NavigationState>;
+};
+type Props = ReduxProps & OwnProps;
```

Comment:
As long as you're not actually mapping anything from the state, `Props` should be `OwnProps` only (also, you should define your own redux-injected props based on what you need -- the `ReduxProps` you're importing probably isn't what you're going to need)

Reply:
Yep, it's a partial implementation (as part of "Retrieve session information" WIP point)